requirements.txt

```diff
@@ -1,8 +1,8 @@
 anyconfig~=0.10.0
 cachetools~=4.1
 click<9.0
-cookiecutter~=1.7.0
-dynaconf>=3.1.2,<4.0.0
+cookiecutter>=2.1.1
```

Comment:
Should we add an upper bound of `<3`?

Reply:
Yeah, after skimming through [this article](https://iscinumpy.dev/post/bound-version-constraints/), similar things elsewhere and from some recent experiences I think we should revisit the way we pin versions also. The bounds on CalVer packages we have should make sense though: black actually has [exactly the stability policy](https://black.readthedocs.io/en/stable/the_black_code_style/index.html#stability-policy) that you mention (although it doesn't really do "breaking" changes the same way at all). Other packages that are CalVer we have a kind of lagging pin where we upper bound to the last version we saw it was working.

So for now I would add `<3` just to be consistent with how we do other requirements, even though I think we should reconsider that general policy.

Reply:
> black actually has [exactly the stability policy](https://black.readthedocs.io/en/stable/the_black_code_style/index.html#stability-policy) that you mention

TIL!